zebra/zebra_vty.c

```diff
@@ -2852,6 +2852,69 @@ static int config_write_forwarding(struct vty *vty)
 	return 0;
 }
 
+DEFUN_HIDDEN (show_frr,
+	      show_frr_cmd,
```

Comment:
why hidden?

Reply:
its a secret

Reply:
I was just thinkint it would be fun to have this but not necessarily something that needs to be visible.
A nice little easter egg so to speak